internal/venice-client-common/src/main/java/com/linkedin/venice/schema/SchemaReader.java

```diff
@@ -17,5 +17,16 @@ public interface SchemaReader extends Closeable {
 
   Schema getLatestValueSchema();
 
+  /**
+   * Get the latest value schema id. This may be different from the largest value schema if the superset schema is not
+   * the value schema with the largest id
+   * @return
+   */
   Integer getLatestValueSchemaId();
```

Comment:
Does it have to be Integer and not int?

Reply:
It's already the existing API and I didn't want to change it in this PR causing larger refactorings, etc